Python/getargs.c

```diff
@@ -858,6 +859,10 @@ convertsimple(PyObject *arg, const char **p_format, va_list *p_va, int flags,
         double dval = PyFloat_AsDouble(arg);
         if (PyErr_Occurred())
             RETURN_ERR_OCCURRED;
+        else if (dval > FLT_MAX)
+            *p = (float)INFINITY;
+        else if (dval < -FLT_MAX)
```

Comment:
I would prefer to use FLT_MIN.

Reply:
`FLT_MIN` is the smallest nonnegative float not what we want here.

Reply:
`FLT_MIN` is the smallest nonnegative representable float, so not what we want here.